game/spawnables/quick_math/quick_math.gd

```diff
@@ -12,11 +12,22 @@ signal teleported(position: Vector2)
 
 
 # Called when the node enters the scene tree for the first time.
+
+#generate random num
+
+
 func _ready() -> void:
-	self.quick_math_label.text = self.question
-	self.quick_math_sprite.texture = load("res://assets/sprites/number_balls/%s.png" % self.answer)
-	self.add_to_group("quick_maths")
+
+	#var num: int = randi() % 9
+	self.quick_math_label.text = question
+	quick_math_sprite.texture = load("res://assets/sprites/number_balls_blank/1b.png")
```

Comment:
It's always the same ball now. Is that intentional?

Reply:
Also, this removed putting quick_math in the quick_math node group. Maybe you removed it elsewhere, but I think it's still around.